src/why3_tactic.ml

```diff
@@ -0,0 +1,198 @@
+(** Calling a prover using Why3. *)
+
+open Terms
+open Extra
+open Timed
+open Scope
+
+(** [default_prover] contains the name of the current prover. Note that it can
+    be changed by using the "set prover <name>" command. *)
```

Comment:
Actually, `<name>` should be `<string>`.

Reply:
Fixed in e066f99.